### PR TITLE
Migrate to minimum supported Go 1.20

### LINKED ISSUE #93: Update to Go 1.20 min
It's time to drop 1.19 support, see https://github.com/nspcc-dev/neo-go/issues/2906, https://github.com/nspcc-dev/neo-go/issues/2907 and https://go.dev/doc/go1.20.

### PR DESCRIPTION
**Pull request by AnnaShaleva**
Close #93.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,18 +3,17 @@ module github.com/nspcc-dev/dbft
 go 1.20
 
 require (
-	github.com/nspcc-dev/rfc6979 v0.2.0
-	github.com/pkg/errors v0.8.1
-	github.com/stretchr/testify v1.7.0
+	github.com/nspcc-dev/rfc6979 v0.2.1
+	github.com/pkg/errors v0.9.1
```
Comment (roman-khimov) [inline go.mod:7]: What?! How could it survive here?

Comment (AnnaShaleva) [reply go.mod:7]: It's from https://github.com/nspcc-dev/dbft/pull/98/commits/70747b716e5e01e0ce87b58538d8c3121bc4d611#diff-554210d4f44afafaa3b5a9b5005b0607ed1dc518dd2b48c4344f72e41d8bacfdR51-R52.

Should we remove `rfc6979` dependency completely?

Comment (AnnaShaleva) [reply go.mod:7]: Oh, it's about errors, will remove it.

Comment (AnnaShaleva) [reply go.mod]: Fixed.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**